db/migrate/20200203200855_add_ancestry_to_miq_ae_namespace.rb

```diff
@@ -0,0 +1,40 @@
+class AddAncestryToMiqAeNamespace < ActiveRecord::Migration[5.0]
+  class MiqAeNamespace < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def update_ns_parent(parent)
```

Comment:
Lucty, as discussed you should be able to use the built-in `build_ancestry_from_parent_ids!` method.